scripts/build_wikidata.js

```diff
@@ -235,8 +235,9 @@ function processEntities(result) {
     const meta = _qidMetadata[qid];
     let target = _wikidata[qid];
     let entity = result.entities[qid];
-    let label = entity.labels && entity.labels.en && entity.labels.en.value;
-    let labelmul =  entity.labels && entity.labels.mul && entity.labels.mul.value;
+    const labelEn = entity.labels && entity.labels.en && entity.labels.en.value;
+    const labelMul =  entity.labels && entity.labels.mul && entity.labels.mul.value;
+    let label = labelEn ? labelEn : labelMul;
```

Comment:
I think we’ll just barely get lucky and avoid editing Wikidata to set the English label to what’s in the multilingual label, which is the issue you’re trying to fix here. The logic below is confusing, but essentially we’re trying to avoid going down the code path where we override the Wikidata label with what’s in OSM. Long-term, it would be safer to separate out the tasks of pulling from and pushing to Wikidata, especially since the pushes are so English-centric.